cmd/flightctl/main.go

```diff
@@ -97,6 +96,15 @@ type GetOptions struct {
 	Continue      string
 }
 
+func getConfig() (*Config, error) {
```

Comment:
I will remove the config changes for waiting for other patches in flight like: #99 and #81